recipes/snakedeploy/meta.yaml

```diff
@@ -21,6 +21,7 @@ build:
 requirements:
   host:
     - pip
+    - pytest-runner
```

Comment:
See https://github.com/snakemake/snakedeploy/pull/13#issue-622843185 for details.

Reply:
Makes sense, let us try to keep that in mind for the next release.